deployment/persistent_chains/main-net/main-net-launch-manual.md

```diff
@@ -0,0 +1,263 @@
+# DCL Mainnet Launch Guide
+
+<!-- markdownlint-disable MD029 -->
+This document is a step-by-step guide for DCL Mainnet launch ceremony. It  describes the following stages in details:
+
+* Pre-Ceremony
+* Ceremony
+* Post-Ceremony
+
+Please see [Running a DCLedger Node](../../../docs/running-node.md) to get the general understanding
+of a node setup logic along with requirements for the hardware and operating system.
+
+## I. Pre-Ceremony
+
+The following steps are expected to be done **before** the ceremony.
+
+1. **Configure Validator/Sentry Node**
```

Comment:
Maybe let's clearly say that the steps 1 and 2 must be done for every VN and Sentry, while steps 3,4,5 need to be done just once.